src/log.py

```diff
@@ -0,0 +1,63 @@
+"""
+log
+
+By: Norma Chang
+"""
+import json
+import os.path
+import time
+
+
+class Log:
+    """
+    Logging class.
+
+    Logs data from vehicle.
+    """
+    def __init__(self, vehicle_name):
+        self.file_name = ""
+        self.class_name = ""
+        self._start_time = time.strftime('%b %d %Y %H:%M:%S', time.gmtime())
+        self._start_time = self._start_time.replace(" ", "").replace(":", "")
+        if vehicle_name.lower() == 'follower':
+            self.class_name = vehicle_name
+        elif vehicle_name.lower() == 'leader':
+            self.class_name = vehicle_name
+        else:
+            raise ValueError('Unexpected class name')
+
+        self.file_name = self.class_name + self._start_time + '.log'
+
+
+    def write_to_file(self, dictionary):
+        """
+        Write vehicle data into text file.
+
+        :param: dictionary: vehicle dictionary
+        :type: dictionary
+        """
+        self.vehicle_dict = {
+                'Time': time.time(),
+                'Dictionary': dictionary
```

Comment:
We don't want a dictionary of dictionaries for each log. We want to add time to the same dictionary.

Reply:
same as previous 2 comments